examples/download_reports.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import finite_state_sdk
+import requests
+
+
+def custom_download_report(token, organization_context, report_type="CSV", report_subtype="ALL_FINDINGS", asset_version_id=None, output_filename=None):
```

Comment:
The report_subtype field and report_type currently utilizes a string value. Considering this, would it be advisable to switch to using enums for better consistency and maintainability?

Reply:
Definitely will consider this for the next release